Sample/SceneDelegate.swift

```diff
@@ -47,7 +45,5 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // Use this method to save data, release shared resources, and store enough scene-specific state information
         // to restore the scene back to its current state.
     }
-
-
 }
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Sample/blob/db347568e1ddc4d261e8ffeb82565d58b3969a2b/Sample/SceneDelegate.swift#L49"></span>Files should have a single trailing newline.<br />
<code>trailing_newline</code> <code>SceneDelegate.swift:49</code></del></td>
    </tr>
  </tbody>
</table>